RefRed/preview_config/preview_config.py

```diff
@@ -8,14 +8,7 @@
 
 class PreviewConfig(QtWidgets.QMainWindow):
 
-    system_name = [
-        "instrument_name",
-        "timestamp",
-        "python_version",
-        "platform",
-        "architecture",
-        "mantid_version",
-    ]
+    system_name = ["instrument_name", "timestamp", "version"]
```

Comment:
is the `mantid_version` info irrelevant?

Reply:
I added it. I also added the `<generator>` tag, which I thought was useful.